app/frontend/src/features/communities/CommunityPage/JoinCommunityButton.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, Story } from "@storybook/react";
+import { mockedService } from "stories/serviceMocks";
+import community from "test/fixtures/community.json";
+
+import JoinCommunityButton from "./JoinCommunityButton";
+
+export default {
+  component: JoinCommunityButton,
+  title: "Communities/CommunityPage/JoinCommunityButton",
+} as Meta;
+
+const Template: Story<any> = (args) => {
```

Comment:
Type the `any` here since you are using `args` with `succeed` and `joined` so it's known what the type is?